src/cascadia/TerminalSettingsModel/TerminalSettings.cpp

```diff
@@ -287,6 +287,11 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             const til::color colorRef{ profile.TabColor().Value() };
             _TabColor = static_cast<winrt::Microsoft::Terminal::Core::Color>(colorRef);
         }
+        std::unordered_map<hstring, uint32_t> featMap{ { L"subs", 1 } };
+        _FontFeatures = single_threaded_map<hstring, uint32_t>(std::move(featMap));
+
+        std::unordered_map<hstring, int64_t> axesMap{ { L"wght", 900 }, { L"slnt", 70 } };
+        _FontAxes = single_threaded_map<hstring, int64_t>(std::move(axesMap));

```

Comment:
This is just here for testing, it'll be removed once the json parsing part of the PR is in